test/integration/profile_security_question_test.go

```diff
@@ -0,0 +1,40 @@
+package integration
+
+import (
+    "context"
+    "fmt"
+    "testing"
+
+    "github.com/stretchr/testify/require"
+    "golang.org/x/exp/slog"
+)
+
+// Assuming `testingMode` and `recorder.ModeReplaying` are defined somewhere in your codebase
+
+func warnSensitiveTest(t *testing.T) {
```

Comment:
Thank you for implementing this! Would it also make sense to call it from the invoice and support ticket tests?